lib/Quantization/Quantization.cpp

```diff
@@ -658,8 +658,7 @@ class FunctionQuantizer : public FunctionConverter {
       auto *dequantize =
           llvm::dyn_cast<DequantizeNode>((*val.getUsers().begin()).getUser());
       TypeRef outTy = val.getType();
-      auto name =
-          NodeValue::generateNodeOutputName(dequantize->getName(), outNum);
+      auto name = NodeValue::generateNodeOutputName(dequantize->getName(), 0);
```

Comment:
nit: Can we actually just use `dequantize->getResult().generateNodeOutputName()` here? I've been trying to move away from uses of this older static version.

Reply:
ah, didn't realize that was there. updated. thx